### PR TITLE
Vanilla js lightbox library

### PR DESCRIPTION
**Pull request by AkkarinForest**
The library: https://github.com/imgix/luminous

### CODE REVIEW

**Comment by gitjul** [comment]
closing in favour of https://github.com/HabitatMap/AirCasting/pull/294, decided to go with a different library

**Comment by gitjul** [comment]
back to this library cos I encountered some problems and decided to overcome them by avoiding them :D

File: app/javascript/packs/elm.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ import "tippy.js/themes/light-border.css";
 import { createObserver } from "../createObserver.js";
 import "../../assets/stylesheets/vendor/jquery-ui-1.8.17.custom.css";
 import "../../assets/stylesheets/vendor/jquery.autocomplete.css";
-import "../../assets/stylesheets/vendor/jquery.lightbox-0.5.css";
+import "../../../node_modules/luminous-lightbox/dist/luminous-basic.css";
```
Comment (3v0k4) [inline app/javascript/packs/elm.js]: Is there a way to import with an absolute name (eg `import "luminous-basic.css")? This seems a bit fragile

Comment (gitjul) [reply app/javascript/packs/elm.js]: that's the way from the documentation..

**Pull request merged**